project/client/js/inventory.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Created by vova on 29.09.2015.
+ */
+Session.set("prob", "");
+var problems = [];
+Template.inventory.events({
+
+    "click #cancel": function(){
+        "use strict";
+        location.assign("http://localhost:3000/index.html");
+        location.reload(true);
+    },
+
+    "click .form": function(event){
+        "use strict";
+        if ($(event.target).css("background-color") === "rgb(255, 0, 0)"){
```

Comment:
not sure if I've understood this code right. But good practice is assignment needed css class to element